transformations/multilingual_lexicon_perturbation/transformation.py

```diff
@@ -0,0 +1,100 @@
+import os
+import random
+import string
+import pandas as pd
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+import nltk
```

Comment:
This should move into the constructor.

Reply:
Is this one for the `nltk.download('punkt')`? I have moved the download to the constructor, let me know if you are referring to the other line of code

Reply:
Yes, I meant the download.